src/code/z_actor.c

```diff
@@ -1714,12 +1714,18 @@ void func_8002F7A0(PlayState* play, Actor* actor, f32 arg2, s16 arg3, f32 arg4)
     func_8002F758(play, actor, arg2, arg3, arg4, 0);
 }
 
-void func_8002F7DC(Actor* actor, u16 sfxId) {
-    Audio_PlaySfxGeneral(sfxId, &actor->projectedPos, 4, &gSfxDefaultFreqAndVolScale, &gSfxDefaultFreqAndVolScale,
-                         &gSfxDefaultReverb);
+/**
+ * Play a sfx at player's position
+ */
+void Player_PlaySfx(Player* player, u16 sfxId) {
```

Comment:
at first I wasnt sure about "play at players position" considering some dont do that. But its probably fine to say the cases that arent using players positions are just hacks/incorrect 

---

I think I would prefer "play a sound effect" instead of "play a sfx" personally. if you want `sfx` to be present you can squeeze in `sfxId` somehow, or use the arg descriptions. But I also think its fine without (personally)

we have one case of `play a sfx` in z_play that we can also change over to be consistent in some other pr

Reply:
I suppose `player's position` is referring to the argument `player` instead of the more generic `player`, so it will always play at the position of arg0 if you will. Not sure if it's worth having `@player` and `@sfxId` doxygen args for trivial cases like this, but maybe it could help clear that up.

`Play a sound effect` is fine, I see the one in z_play but there are many other cases of `sfx` -> `sound effect` in comments that could be changed. So I can defer those to a separate cleanup PR so they don't get hidden in the large number of file changes, and I'll just adjust these comments for now.

Reply:
yea I dont care for the doxygen argument descriptions in this case personally. 

I think sfx makes sense when referring to the system, or an sfx id specifically, but yeah I think when saying the literal sentence "play a sound effect", its better to say the whole thing instead of the abbreviation